app/views/client_matches/index.haml

```diff
@@ -30,8 +30,7 @@
 %ul.nav.nav-tabs
   - @possible_statuses.each do |label, status|
     %li.nav-item{class:('active' if status == @status)}
-      - # FIXME replace permit! with a whitelist and remove from brakeman.ignore
-      = link_to url_for(params.permit!.merge(status: status, page: nil)), class: 'nav-link' do
+      = link_to url_for({status: status}), class: 'nav-link' do
```

Comment:
This looks like we lost quite a bit.  Should there be a merge with something here?  I'm guessing it's trying to preserve sort order.

Reply:
This switches between the different tabs in Duplicate Clients processing, and there doesn't seem to be any sorting/filtering.